### DISCUSSION TITLE
(1.0.4): chia_wallet takes constantly >80% cpu (>80% of one core) when farming; is it normal?

### DISCUSSION THREAD

**Discussion by ardabro** [Chia-Network/chia-blockchain#1880] [Q&A]
This is my typical cpu usage during farming. Is it normal that chia_wallet consumes so much all **the time**?

`  PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND`
`12503 arek      20   0  496648 117544  20676 R  84.9   3.1  41:38.57 chia_wallet`
`12502 arek      20   0 1013688 542452  20432 S  17.0  14.2 105:50.60 chia_full_node`
` 1263 root      20   0       0      0      0 S   0.7   0.0   3:22.05 btrfs-transacti`
`12288 arek      20   0  215856  58784  16512 S   0.6   1.5   0:29.08 chia_daemon`
`12500 arek      20   0  302976  56096  17920 S   0.4   1.5   0:27.85 chia_harvester`

**Comment by dorkmo** [comment]
you could do `chia stop wallet` really its not neccessary to have on

**Reply by ardabro** [reply]
But it starts when I invoke `chia start farmer`. If I don't need it for farming what exactly does it do?

**Comment by hoffmang9** [comment]
Wallet only lets you see and spend coins locally.

**Reply by ardabro** [reply]
Oh man! When I stopped wallet it stopped full node as well and stopped farming!

```
(venv) #screen#>chia stop wallet
chia_wallet: Stopped
chia_full_node: Stopped
(venv) #screen#>../farm_status 
================================================================================ Wed 14 Apr 21:01:28 CEST 2021
Connection error. Check if wallet is running at 9256
Connection error. Check if full node is running at 8555
Farming status: Not available
Total chia farmed: Unknown
User transaction fees: Unknown

```

**Reply by dorkmo** [reply]
my bad! should have been `chia stop wallet-only` i believe

**Reply by dorkmo** [reply]
you can type in `chia stop --help` and itll show you the options

**Comment by fullet-tortuga** [comment]
The thing is, is it actually doing anything that justifies that cpu usage?
